core/client/hammer/hammer.go

```diff
@@ -76,125 +96,187 @@ func New(ctx context.Context, dial DialFunc, callOptions CallOptions,
 		timeout:     timeout,
 		ktCli:       ktCli,
 		appID:       fmt.Sprintf("hammer_%v", time.Now().Format("2006-01-02/15:04:05")),
-		config:      config,
+		domain:      domain,
 
 		signers:        []*tink.KeysetHandle{keyset},
 		authorizedKeys: authorizedKeys.Keyset(),
 	}, nil
 }
 
-// Run runs operationCount with up to maxWorkers over ramp duration.
-func (h *Hammer) Run(ctx context.Context, operationCount, maxWorkers int, ramp time.Duration) {
-	jobs := gen(ctx, operationCount)
-	outputs := h.launchWorkers(ctx, jobs, maxWorkers, ramp)
-	h.printStats(1*time.Second, outputs)
-}
+// Run runs operationCount with up to maxWorkers.
```

Comment:
numWorkers?
Is the "up to" bit really "exactly"? I couldn't find when less is possible.

Reply:
It was an old comment. Updated.